sale_exception/models/sale.py

```diff
@@ -53,11 +53,12 @@ def _check_sale_check_exception(self, vals):
         if check_exceptions:
             self.sale_check_exception()
 
-    @api.model
```

Comment:
Hi thanks for the migration.

It's not related to your change but after think twice
I think that the check on create is useless !

Indeed if you look at the method sale_check_exception, it explicitly skip all sale order that are not in the state "sale", when we create a sale order the state is always "draft".

So indeed of adding some extra complexity to the code, maybe it's a good opportunity to simplify it and just to remove the inherit on "def create".

I should have seen it before but I miss to review this PR : https://github.com/OCA/sale-workflow/pull/916

@simahawk @rousseldenis @sbejaoui what do you think ?

Reply:
@florian-dacosta

Reply:
I agree the create override can be removed as it seems useless.